.github/workflows/main.yml

```diff
@@ -4,6 +4,7 @@ on: pull_request
 
 env:
   GIT_CONFIG_PARAMETERS: "'checkout.workers=56'"
+  MSYSTEM: MINGW64
```

Comment:
Since we're not setting `MSYSTEM=MINGW32` explicitly in the 32-bit jobs, that's what causes [this problem](https://github.com/git-for-windows/build-extra/actions/runs/3625271670/jobs/6122096351#step:5:10).